cdserver/features/authentication/step_definitions/authentication_steps.rb

```diff
@@ -1,10 +0,0 @@
-Given(/^The following user exists in the system$/) do |table|
```

Comment:
this should check to see if the user exists, and add them if they don't already exist, shouldn't it?